app/src/test/java/com/fstyle/structure_android/data/repository/UserRepositoryTest.java

```diff
@@ -1,11 +1,59 @@
 package com.fstyle.structure_android.data.repository;
 
+import android.database.sqlite.SQLiteDatabase;
+import com.fstyle.structure_android.data.model.User;
+import com.fstyle.structure_android.data.source.local.UserLocalDataSource;
+import com.fstyle.structure_android.data.source.remote.UserRemoteDataSource;
+import com.fstyle.structure_android.data.source.remote.config.service.NameApi;
+import com.google.common.collect.Lists;
+import io.reactivex.subscribers.TestSubscriber;
+import java.util.List;
+import org.junit.After;
+import org.junit.Before;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
 
 /**
  * Created by daolq on 12/22/17.
+ * Class Test for ({@link UserRepository})
  */
 public class UserRepositoryTest {
-    public void searchUsers() throws Exception {
+
+    private static final String LIMIT_NUMBER = "3";
+    private static final String KEYWORK = "abc";
+
+    private static final User user = new User("Name 1");
```

Comment:
ERROR: Static final field names must either be all caps (e.g. int HEIGHT_PX) for true constants, or start with s (e.g. AtomicInteger sNextId or Runnable sSuspendTask) for fields with mutable state or that dont feel like constants.